content/content-tool/tool/src/java/org/sakaiproject/content/tool/ListItem.java

```diff
@@ -1642,7 +1642,10 @@ protected void captureDescription(ParameterParser params, String index)
 		if(description != null)
 		{
 			StringBuilder errorMessages = new StringBuilder();
-			description = FormattedText.processFormattedText(description, errorMessages);
+						
+			// This process delete line breaks
+			//description = FormattedText.processFormattedText(description, errorMessages);
```

Comment:
Just because line breaks are being removed does it make sense to not screen the rest of the text?

If no, then why not just adjust formatted text to allow line breaks?

Reply:
I have modified antisamy directive to allow line breaks.